scripts/ci/cleanup-deployment.sh

```diff
@@ -1,3 +1,5 @@
-#! /bin/bash
+#!/usr/bin/env bash
 
-kubectl -n stackrox get cm,deploy,ds,networkpolicy,pv,pvc,secret,svc,serviceaccount -o name | xargs kubectl -n stackrox delete --wait
+namespace=${1:-stackrox}
```

Comment:
qq here, i'm sure you've thought about this already: where do we call this script? Could we pass multiple namespaces to it?

Reply:
As far as I can tell it is only called [here](https://github.com/stackrox/stackrox/blob/9bb2106bda7b729bf7c8d19b2ba559ce328e9f61/scripts/ci/gke.sh#L324).

Surely it could be generalized to multiple namespaces -- I just didn't have the requirement as part of scanner v4 project.

It seems to me that there is a lot of room for streamlining in the collection of scripts within our codebase...